urbit/umbrel-app.yml

```diff
@@ -35,24 +35,14 @@ torOnly: false
 submitter: ~mopfel-winrux
 submission: https://github.com/getumbrel/umbrel/pull/1246
 releaseNotes: >-
-  -   Revamped Urbit’s terminal stack. This release contains the first
-      phase of a longer-running project towards a next-generation terminal
-      experience. Highlights in this release include:
-      
-      -   Support for 2D cursor movement, and by extension 2D screen
-          drawing.
-          
-      -   Minimal mouse-click support.
-      
-      -   Faster pasting of large texts.
-      
-      -   Informal printfs (~&) now get rendered in grey, to visually
-          distinguish them from formal dill output.
-          
-  -   Updated versioning scheme for pre-release binaries.
-  
-  
-  -   The code for the Vere runtime has been moved out of the urbit/urbit
-      repository, and now lives in urbit/vere.
-
-  Full release notes here: https://github.com/urbit/vere/releases/tag/vere-v1.17
+  Adds runtime support for the doccords system, which is documentation
+  parsed from comments in source code (Hoon’s version of docstrings).
+  Shoutout to ~datnut-pollen (@drbeefsupreme) for doing the huge amount of
```

Comment:
Due to usage of `>-` with this section. An empty line is needed here to ensure this ends up on a new line and not appended to the previous line.

I would also suggest a double newline to get empty space between sections, eg where you have the lines `This release accompanies` and `Full release note`.

This is a screenshot of how these notes appear atm:
![Screenshot 2023-02-02 at 01 34 26](https://user-images.githubusercontent.com/1112574/216209353-5adb359e-5522-434e-8d9f-a73f691918a1.png)

Reply:
No problem, let me know if this looks better